integration_tests/steps/infinite_bus_tests.py

```diff
@@ -37,7 +37,7 @@ def check_buy_behaviour_ib(context):
         for trade in core_stats[name_uuid_map['Grid']]['trades']:
             assert limit_float_precision(trade['energy_rate']) <= \
                    core_stats[name_uuid_map['Infinite Bus']]['energy_rate'] or \
```

Comment:
Please give following a try to avoid flake8

```suggestion
            assert (limit_float_precision(trade['energy_rate']) <=
                    core_stats[name_uuid_map['Infinite Bus']]['energy_rate']) \
                   or "Infinite Bus" is not trade['seller']
```

Reply:
Unfortunately, this doesn't seem to fix the problem: flake8 is still complaining about the comparison operator:
> F632 use ==/!= to compare str, bytes, and int literals

Reply:
Ignore me, I was testing with an old flake version.